server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/EmailQueryMethodContract.scala

```diff
@@ -292,7 +300,50 @@ trait EmailQueryMethodContract {
         .body
         .asString
 
-      val responseSecondCall = `given`
+    assertThatJson(response)
+      .inPath("$.methodResponses[0][1].ids")
+      .isEqualTo(s"""["${messageId3.serialize()}", "${messageId2.serialize()}", "${messageId1.serialize()}"]""")
+    }
+  }
+
+  @Test
+  def listMailsShouldBeSortedASCByArrivalAtWithComparator(server: GuiceJamesServer): Unit = {
```

Comment:
```suggestion
  def listMailsShouldBeSortedInAscendingOrder(server: GuiceJamesServer): Unit = {
```
Add the descending case